src/python/CMakeLists.txt

```diff
@@ -484,6 +486,9 @@ if(PYTHONINTERP_FOUND)
       add_gudhi_py_test(test_dtm)
     endif()
 
+    # Weighted Rips
+    add_gudhi_py_test(test_weighted_rips)
```

Comment:
To be safe, protect this line with `if(SCIPY_FOUND)` and `endif()`? I don't remember if scipy is mandatory or optional in gudhi, and the protection can't hurt.